packages/next/package.json

```diff
@@ -102,6 +102,7 @@
   },
   "peerDependencies": {
     "@opentelemetry/api": "^1.1.0",
+    "lightningcss": "^1.22.1",
```

Comment:
I think we want to remove dependencies to this package (https://github.com/vercel/next.js/commit/1f6feac2d4314ac14c875e47b88bf84be3cec9cd)